extensions/who-is-visiting-this-content/www/styles.css

```diff
@@ -0,0 +1,52 @@
+/* Styles used by the Most Used Content table */
+.metrics-tbl {
+  font-size: 0.875rem;
+  line-height: 1.125rem;
+}
+
+.metrics-tbl a {
+  color: inherit;
+  text-decoration: none;
+}
+
+.metrics-tbl a:hover,
+.metrics-tbl a:focus {
+  text-decoration: underline;
+  text-decoration-thickness: max(1px, 0.0625rem);
+}
```

Comment:
I tried looking for anchors inside of a `metrics-tbl` I may have missed it though.

Reply:
The little "open" up-right arrow icons are anchors. The second block here isn't needed though.